extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -57,6 +57,8 @@ export class DeployConfigPage extends DacFxConfigPage {
 	async onPageEnter(): Promise<boolean> {
 		let r1 = await this.populateServerDropdown();
 		let r2 = await this.populateDeployDatabaseDropdown();
+		// get existing database values to verify if new database name is valid
+		await this.getDatabaseValues();
```

Comment:
Shouldn't this happen already when the server dropdown is populated (and thus a new server is "selected")? If not that seems like a bug

Reply:
It is, but this is so the database values get refreshed when going back and forth between pages so that the list of database names are more current in case a db got created while the wizard is open. Does it seem unnecessary to have this?

Reply:
Ah, ok. I'm not sure it's hugely necessary since we already have a possibility of stale data if the user just sits on the page for a while but I don't have a problem with doing it either.